<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharWindow</class>
 <widget class="QWidget" name="CharWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEditContent"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditMes"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSendMes">
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSendFile">
       <property name="text">
        <string>发送文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
